Makefile

```diff
@@ -63,6 +64,13 @@ staticcheck: $(STATICCHECK)
 	@echo ">> running staticcheck"
 	@$(STATICCHECK) $(pkgs)
 
+goveralls: cover $(GOVERALLS)
+ifndef COVERALLS_TOKEN
+	$(error COVERALLS_TOKEN is undefined)
```

Comment:
Another documentation request: To run `make goveralls’, it looks like I would need to set COVERALLS_TOKEN. What would I set it to?